hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOffloadableSetUnlockBackupOperation.java

```diff
@@ -0,0 +1,175 @@
+
+import static com.hazelcast.core.EntryEventType.ADDED;
+import static com.hazelcast.core.EntryEventType.REMOVED;
+import static com.hazelcast.core.EntryEventType.UPDATED;
+import static com.hazelcast.map.impl.EntryViews.createSimpleEntryView;
+import static com.hazelcast.map.impl.MapService.SERVICE_NAME;
+
+/**
+ * Set & Unlock processing for the EntryOffloadableBackupOperation
+ *
+ * See the javadoc on {@link EntryOffloadableOperation}
+ */
+public class EntryOffloadableSetUnlockBackupOperation extends UnlockBackupOperation {
+
+    protected String name;
+    protected Data value;
+    protected Data oldValue;
+    protected long begin;
+    protected EntryEventType modificationType;
+
+    protected transient MapServiceContext mapServiceContext;
+    protected transient MapService mapService;
+    protected transient MapContainer mapContainer;
+    protected transient RecordStore recordStore;
+    protected transient MapEventPublisher mapEventPublisher;
+    protected transient Invalidator nearCacheInvalidator;
+
+    public EntryOffloadableSetUnlockBackupOperation() {
+    }
+
+    public EntryOffloadableSetUnlockBackupOperation(ObjectNamespace namespace, String name, EntryEventType modificationType,
+                                                    Data key, Data oldValue, Data value, String caller, long threadId,
+                                                    long begin) {
+        super(namespace, key, threadId, caller, false);
+        this.name = name;
+        this.value = value;
+        this.oldValue = oldValue;
+        this.begin = begin;
+        this.modificationType = modificationType;
+    }
+
+    @Override
+    public void beforeRun() throws Exception {
+        super.beforeRun();
+        this.mapService = getNodeEngine().getService(SERVICE_NAME);
+        this.mapServiceContext = mapService.getMapServiceContext();
+        this.mapContainer = mapService.getMapServiceContext().getMapContainer(name);
+        this.mapEventPublisher = mapService.getMapServiceContext().getMapEventPublisher();
+        this.recordStore = mapServiceContext.getRecordStore(getPartitionId(), name);
+        this.nearCacheInvalidator = mapServiceContext.getMapNearCacheManager().getInvalidator();
+    }
+
+    @Override
+    public void run() throws Exception {
+        updateRecordStore();
+        unlockKey();
+    }
+
+    private void updateRecordStore() {
+        if (modificationType == REMOVED) {
+            recordStore.removeBackup(key);
+        } else if (modificationType == ADDED || modificationType == UPDATED) {
+            recordStore.putBackup(key, value);
+        } else {
+            throw new IllegalArgumentException("Unsupported event type " + modificationType);
+        }
+    }
+
+    private void unlockKey() {
+        LockStoreImpl lockStore = getLockStore();
+        response = lockStore.unlock(key, originalCallerUuid, threadId, getReferenceCallId());
+        lockStore.pollExpiredAwaitOp(key);
```

Comment:
`pollExpiredAwaitOp()` is for `ICondition`. `IMap` doesn't support conditions. So, you should just call `recordStore.unlock(...)` instead of extending `c.h.c.lock.operations.UnlockBackupOperation`. Similar to `TxnUnlockBackupOperation`.